CMakeLists.txt

```diff
@@ -118,4 +118,4 @@ if(LY_INSTALL_ENABLED)
     ly_setup_o3de_install()
     # 9. CPack information (to be included after install)
     include(cmake/Packaging.cmake)
-endif()
+endif()
```

Comment:
uber nit: Can this change be undone please (looks like it removed the EOL character which we'd like to keep 🙂)